Interchange: updates (#578)
cosmos/sdk-tutorials: interchain-exchange/tutorial/09-walkthrough.md

## Configure the relayer

> Configure the relayer between `Mars` and `Venus` blockchains. Use the `--advanved` flag to pass custom port name and version. For `Mars` use the default ports, for `Venus` ports are specified in `venus.yml`. IBC port and version is defined in the source code of the module.

Reviewer: ```suggestion
Configure the relayer between `Mars` and `Venus` blockchains. Use the `--advanced` flag to pass custom port name and version. For `Mars` use the default ports, for `Venus` ports are specified in `venus.yml`. IBC port and version is defined in the source code of the module.
```

> Configure the relayer between `Mars` and `Venus` blockchains. We'll be using the `--advanved` flag to pass custom port name and version. For `Mars` we're using the default ports, for `Venus` ports are specified in `venus.yml`. IBC port and version is defined in the source code of the module.

Reviewer: ```suggestion
Configure the relayer between `Mars` and `Venus` blockchains. Use the `--advanved` flag to pass custom port name and version. For `Mars` use the default ports, for `Venus` ports are specified in `venus.yml`. IBC port and version is defined in the source code of the module.
```

## Create an Orderbook

<!-- Create an orderbook for a new pair of token to the exchange. 
Source blockchain is `Mars`, target blockchain is `Venus`.

The orderbook is to sell `mcx` and buy `vcx` token. -->

> Let's create orderbooks for a pair of tokens.

Reviewer: As recent standards please don't use "we" or "let us" but directly address the reader "you".

> Let's create orderbooks for a pair of tokens.

Reviewer: ```suggestion
Create orderbooks for a pair of tokens.
```

```bash
# create the pair
interchanged tx ibcdex send-createPair ibcdex channel-0 mcx vcx --from alice --chain-id mars --home ~/.mars
```

> `send-createPair` command will create, sign and broadcast a transaction containing a `MsgSendCreatePair` message. The command accepts 4 arguments: port name (`ibcdex`) defined in the source code, channel name (`channel-0`) created by the module and two denom strings. We also need to specify chain ID and a custom home directory.

Reviewer: ```suggestion
`send-createPair` command will create, sign and broadcast a transaction containing a `MsgSendCreatePair` message. The command accepts 4 arguments: port name (`ibcdex`) defined in the source code, channel name (`channel-0`) created by the module and two denom strings. Specify chain ID and a custom home directory.
```
